qiskit_nature/problems/second_quantization/electronic/electronic_structure_problem.py

```diff
@@ -136,3 +138,32 @@ def filter_criterion(self, eigenstate, eigenvalue, aux_values):
                 num_particles_aux) and np.isclose(0., total_angular_momentum_aux)
 
         return partial(filter_criterion, self)
+
+    def symmetry_sector_locator(self, z2_symmetries: Z2Symmetries) -> Optional[List[int]]:
+        """Given the detected Z2Symmetries can determine the correct sector of the tapered
+        operators so the correct one can be returned
+
+        Args:
+            z2_symmetries: the z2 symmetries object.
+
+        Returns:
+            the sector of the tapered operators with the problem solution
```

Comment:
Some docs have dots at the end and some do not. Would be good to make it consistent.